maven/impl-maven/pom.xml

```diff
@@ -67,26 +67,22 @@
             <artifactId>plexus-classworlds</artifactId>
             <version>${version.org.codehaus.plexus-classworlds}</version>
         </dependency>
-        <dependency>
```

Comment:
And same with plexus-component-annotations please

Reply:
Unfortunately no, as it's deprecated / removed due to other upgrades